schemas/error-schema.json

```diff
@@ -71,6 +71,11 @@
               "description": "Handling call stack",
               "readOnly": true
             },
+            "source_type": {
+              "type": "string",
```

Comment:
should we maybe have enum here? this way audience will be notified about every new language we support